                
*JUMP v1.0*

 
Table of Contents:

 
Introduction

 
Download

 
Usage

 
Parameter guide

   
*Introduction*

 
JUMP (Jumbo Mass
Spectrometry-based Proteomics Tool)
is a new database search engine for peptide identification. JUMP incorporates
sequence tag generation and tag scoring into database search.

 
*Download*

 
The source code can be downloaded *here*
and used according to the terms of the GNU General Public License (GPL),
version 2 or later. The parameter file can be downloaded *here*. 

 
Sample data can be downloaded: data set 1 for regular
identification and data set 2 for post-translational modifications.

 
*Usage*

 
The basic
usage of JUMP is quite simple:

 
JUMP.pl -param &lt;JUMP
parameter file&gt; -rawfile &lt; MS/MS data file&gt;

 
-param
&lt;file&gt;             
specifies a JUMP parameter file

 
-rawfile
&lt;file&gt;             
specifies an MS/MS data file

 
An MS/MS data file can be
either .RAW or mzXML file that can be converted from .RAW file by various
software, such as ReAdW or msconverter.

   
*Parameters guide*
                                                                                                                                                                                                                                                                       
|| *Name* || *Description* || *Type* || **Default** ||
|| database_name || Specifies the FASTA or index protein database for search. If a FASTA  file was specified, the program will automatically build an indexed  database   || string || None ||

   
############################## Parameter file for JUMP
#################################

 
# JUMP version 10.02 

 
# Date: 02/01/2013 

 
# Author: Xusheng Wang 

 
####################################################################################

   
############################## Database
#############################################

 
database_name = /home/xwang4/JUMP_database/human/human_ft_mc2_c57.fasta.mdx

 
enzyme_info = Tryptic KR P

 
digestion = full

 
max_mis_cleavage = 2

 
min_peptide_mass = 400.0000

 
max_peptide_mass = 6000.0000

 
inclusion_decoy = 1

 
############################## Preprocessing ##########################################

 
precursor_mass_window = 0.19

 
intrascanppm = 10

 
interscanppm = 5

 
preproc_msscanrange = 4

 
peptide_tolerance = 15

 
peptide_tolerance_units = 2        #
1 = Th; 2 = ppm;

 
vary_tolerance = 0

 
peptide_tolerance_frac = 0.8

 
ms2_consolidation = 6

 
prec_window = 3

 
ppm = 30

 
M_M1 = 0.3

 
Mn_Mn1 = 1

 
charge12_ppm = 30

 
#############################
Tag##################################################

 
tag_tolerance = 0.02

 
low_mass = 57

 
high_mass = 187

 
tag_select_method = rank_p  # method can be input: hyper_p or
rank_p

 
######################Search
###########################################

 
ion_series = 0 1 0 0 0 0 0 1 0  # a, b, c, d, v, w, x, y, and z
ions respectively

 
frag_mass_tolerance = 0.02

 
tag_search_method = 2       # 1. last when
found 2. search using the number of tag assigned by 'max_number_tag_for_search'

 
max_number_tags_for_search = 500  # If this number larger than the
total number of tag, the total number of tag will be used

 
tag_coeff_evalue = 4

 
##################### Dynamic Modification
######################################

 
# C:  57.02146 carbamidomethylation or 71.0371 acrylamide

 
# STY: 79.96633

 
# M: 15.99492

 
# GG: 114.04293

 
# SILAC K:4.02511, 6.02013, 8.01420

 
# SILAC R:6.02013, 10.00827

 
#STY = 0.000   

 
#K = 0.000 

 
dynamic_M = 15.99492

 
max_modif_num = 6

 
#################### Static Modification
############################################

 
add_Cterm_peptide = 0.0000

 
add_Nterm_peptide = 0.0000

 
add_G_Glycine = 0.0000

 
add_A_Alanine = 0.0000

 
add_S_Serine = 0.0000

 
add_P_Proline = 0.0000

 
add_V_Valine = 0.0000

 
add_T_Threonine = 0.0000

 
add_C_Cysteine = 57.02146

 
add_L_Leucine = 0.0000

 
add_I_Isoleucine = 0.0000

 
add_X_LorI = 0.0000

 
add_N_Asparagine = 0.0000

 
add_O_Ornithine = 0.0000

 
add_B_avg_NandD = 0.0000

 
add_D_Aspartic_Acid = 0.0000

 
add_Q_Glutamine = 0.0000

 
add_K_Lysine = 0.0000

 
add_Z_avg_QandE = 0.0000

 
add_E_Glutamic_Acid = 0.0000

 
add_M_Methionine = 0.0000

 
add_H_Histidine = 0.0000

 
add_F_Phenylalanine = 0.0000

 
add_R_Arginine = 0.0000

 
add_Y_Tyrosine = 0.0000

 
add_W_Tryptophan = 0.0000

 
add_J_user_amino_acid = 0.0000

 
add_U_user_amino_acid = 0.0000

 
#################### Cluster system
####################################################

 
cluster = 1

 
Job_Management_System = SGE

 
#################### Simulation ######################################################

 
simulation = 0   # 1 = yes; 0 = no

 
sim_MS1 = 0    # unit is ppm

 
sim_MS2 = 0     # unit is Da

 
###################### Output for results
############################################### 

 
number_of_selected_result = 3

 
number_of_detailed_result = 100

 
#####################################################################################

   
JUMP manual

 
*System Requirement*

 
JUMP must meet these minimum requirements.

                  
|| 
*System*

 || 
*Requirements*

 ||
|| 
Hardware

 || 
Cluster settings:

 
·         Run on a cluster system

 
&Oslash; 
  SGE or LSF job management system

 
&Oslash; 
  32 GB memory on each node

   
·         Run on a single server 

 
&Oslash; 
  32 GB memory

 
&Oslash; 
  2 GHz processor

   ||
|| 
Software

 || 
WINE install if run .raw file

 
Perl modules:

   ||
||   ||   ||
   
*Features*

 
JUMP includes the following features:

   
1.    Support for input
formats: .raw or .mzXML

 
2.    Support for FASTA
databases or indexed databases

 
3.     

            