
*JUMP**v1.0*


Table
of Contents:


Introduction


Download


Usage


Parameter
guide

 
*Introduction*


JUMP (Jumbo Mass Spectrometry-based
Proteomics Tool) is a new database
search engine for peptide identification. JUMP
incorporates sequence tag generation and tag scoring into database search.


*Download*


The source code can be downloaded *here* and used
according to the terms of the GNU General Public License (GPL), version 2
or later. The parameter file can be downloaded *here*. 


Sample
data can be downloaded: data set 1 for regular identification and data set 2
for post-translational modifications.


*Usage*


The basic usage of JUMP is
quite simple:


JUMP.pl -param &lt;JUMP
parameter file&gt; -rawfile &lt; MS/MS data file&gt;


-param
&lt;file&gt;             
specifies a JUMP parameter file


-rawfile
&lt;file&gt;             
specifies an MS/MS data file


An MS/MS data file can be
either .RAW or mzXML file that can be converted from .RAW file by various
software, such as ReAdW or msconverter.

 
*Parameters
guide*


{|  class=MsoTableGrid border=1 cellspacing=0 cellpadding=0 width=655 style='width:491.4pt;border-collapse:collapse;border:none' 
|  width=98 valign=top style='width:73.45pt;border:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
*Category*


|  width=149 valign=top style='width:111.95pt;border:solid windowtext 1.0pt;  border-left:none;padding:0in 5.4pt 0in 5.4pt' | 
*Name*


|  width=245 valign=top style='width:183.75pt;border:solid windowtext 1.0pt;  border-left:none;padding:0in 5.4pt 0in 5.4pt' | 
*Description*


|  width=63 valign=top style='width:47.25pt;border:solid windowtext 1.0pt;  border-left:none;padding:0in 5.4pt 0in 5.4pt' | 
*Type*


|  width=100 valign=top style='width:75.0pt;border:solid windowtext 1.0pt;  border-left:none;padding:0in 5.4pt 0in 5.4pt' | 
*Default**


|- 
|  width=98 rowspan=7 style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Database


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
database_name


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the FASTA or index protein database for search. If a FASTA
file was specified, the program will automatically build an indexed
database  


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
string


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
None


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
enzyme_info


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the enzyme information used for creating a database 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
string


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Tryptic KR P


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
digestion


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the digestion methods. Three types of methods can be used,
including Full, Partial, and None


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
string


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Full


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
max_mis_cleavage


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
A missed cleavage is defined as a site within the peptide matching
one of the cleavage rules (see enzyme_info). This parameter is used to set
the maximum number of missed cleavages.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
2


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
min_peptide_mass


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the minimum mass value of peptides in the database


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
400


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
max_peptide_mass


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the maximum mass value of peptides in the database


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
6000


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
inclusion_decoy


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
This parameter is to set whether the database contains the decoy
sequences


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
boolean


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1 (true)


0 (false)


|-  style='height:27.35pt'
|  width=98 rowspan=11 style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
Preprocessing


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
isolation_window


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
Specifies the isolation window used for peak selection, with unit of
m/z


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
1.0


|-  style='height:27.35pt'
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
intrascanppm


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
ppm for peak matching within the same scan


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:27.35pt' | 
10


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
interscanppm


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
ppm for peak matching between different scans


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
5


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
preproc_msscanrange


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
range of ms scans used for decharging


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
4


|-  style='height:28.3pt'
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:28.3pt' | 
ms2_consolidation


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:28.3pt' | 
Specifies number of peaks to keep within 100 m/z units  


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:28.3pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt;height:28.3pt' | 
6


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
ms2_deisotope


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Speficies whether to perform deisotope for MS2 peaks


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
boolean


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1 (true)


0 (false)


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
prec_mass_window


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies to remove the peaks with the window with precursor mass +/-
value/2 (additional static 0.25 tolerance to either side)


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
3.0


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
ppm


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Tolerance (ppm) used for MS2 deisotope 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
30


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
M_M1


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
The intensity ratio between the peaks and following peaks. 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
0.3


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Mn_Mn1


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
The intensity ratio between the peaks and following peaks.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
charge12_ppm


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies mass tolerance 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
30


|- 
|  width=98 valign=top style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' |  
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
ppi_percentage


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' |  
|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' |  
|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
10


|- 
|  width=98 rowspan=4 style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Tag


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
tag_tolerance


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Mass tolerance used for tag inference


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
0.02 (high)


0.5 (low)


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
low_mass


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
The minimum mass allowed to define a residue


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
57


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
high_mass


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
The maximum mass allowed to define a residue


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
180


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
tag_select_method


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Two tag scoring methods can be used, including rank_p and hyper_p.
The rank_p method calculates the score based on wilcox rank sum method; the
hyper_p calculates the score based on hypergeometric method.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
string


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
rank_p (high)


hyper_p (low)


|- 
|  width=98 rowspan=7 style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Search


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
ion_series


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
 a, b, c, d, v, w, x, y, and z ions respectively. The values
entered for these paramters should either 0 or 1. 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
String


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' |  
|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
peptide_tolerance


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
This parameter is used to find candidate peptides. The candidate
peptides is selected if the candidate mass is within this tolerance of the
experimental precursor mass. The units can be specified by the ‘peptide_tolerance_units’
parameter.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
15


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
peptide_tolerance_units


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Two types of mass units: 1 for Da and 2 for parts per million (ppm).


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
2


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
frag_mass_tolerance


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the fragment mass tolerance. The fragment tolerance should
be set at 0.01 for high depended on high-resolution instrument resolution; it
should be set at 0.5 for low-resolution instrument. 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
0.01 (high)


0.5 (low)


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
tag_search_method


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Two search methods can be specified: 1 for ; 2 for searching by the a
certain number of tags 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1 or 2


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
max_number_tags_for_search


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
This parameter is used to define the number of tags that are used for
search. The tag is sorted in terms of its p-value. The software search


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
100


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
tag_coeff_evalue


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the weight of tag E value for calculating weighted E value.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1.0


|- 
|  width=98 rowspan=3 style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Modification


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
dynamic_AA


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
This parameter is used to define dynamic amino acids. 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' |  
|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
add_AA_name


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
This is static modification. If the value for any residues is setted,
it should always be treated as having a modification on their natural mass. 


For example, the cysteine is carboxymethylated, this parameter would
be set to be: add_C_Cysteine = 57.02146.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
float


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' |  
|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
max_modif_num


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
This parameter is used to set the maximum number of dynamic
modifications that can be in the candidate peptide. 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
6


|- 
|  width=98 rowspan=2 valign=top style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Cluster system


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
cluster


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies whether JUMP will be run on the cluster


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
boolean


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1 (true)


0 (false)


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Job_Management_System


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the platform of the cluster. Three types of jobs can be
used: SGE, PBS and LSF.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
String


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
SGE


|- 
|  width=98 rowspan=2 valign=top style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Ouput


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
number_of_selected_result


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the number of records being shown in the selected output
section in the output file. 


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
3


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
number_of_detailed_result


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies the number of records being shown in the “Selected
identified peptide” section in the output file.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
100


|- 
|  width=98 rowspan=3 valign=top style='width:73.45pt;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Simulation


|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
simulation


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Specifies whether to perform simulation.  If true, either one of
the following methods can used to generate a falsified data sets based on
input real data.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
boolean


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
1 (true)


0 (false)


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
sim_MS1


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
If the ‘simulation’ parameter is set, the data set was generated by
adding precusor ion mass of each MS/MS spectrum by 100 ppm.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
100


|- 
|  width=149 valign=top style='width:111.95pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
sim_MS2


|  width=245 valign=top style='width:183.75pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
If the ‘simulation’ parameter is set, the data set was generated by
randomizing the MS/MS spectrum. Briefly, we generated uniform random number from
-5 to 5, and added this random number to each m/z of each peak.


|  width=63 valign=top style='width:47.25pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
integer


|  width=100 valign=top style='width:75.0pt;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
5


|}
Note: * represents suggested value for JUMP.

        <br> 
*High-resolution data set*

 
############################## Parameter file for JUMP
#################################


# JUMP version 10.02 


# Date: 02/01/2013 


# Author: Xusheng Wang 


####################################################################################

 
############################## Database #############################################


database_name = /home/xwang4/JUMP_database/human/human_ft_mc2_c57.fasta.mdx


enzyme_info = Tryptic KR P


digestion = full


max_mis_cleavage = 2


min_peptide_mass = 400.0000


max_peptide_mass = 6000.0000


inclusion_decoy = 1


############################## Preprocessing ##########################################


precursor_mass_window = 0.19


intrascanppm = 10


interscanppm = 5


preproc_msscanrange = 4


peptide_tolerance = 15


peptide_tolerance_units = 2        #
1 = Th; 2 = ppm;


vary_tolerance = 0


peptide_tolerance_frac = 0.8


ms2_consolidation = 6


prec_window = 3


ppm = 30


M_M1 = 0.3


Mn_Mn1 = 1


charge12_ppm = 30


############################# Tag##################################################


tag_tolerance = 0.02


low_mass = 57


high_mass = 187


tag_select_method = rank_p  # method can be input: hyper_p or
rank_p


######################Search
###########################################


ion_series = 0 1 0 0 0 0 0 1 0  # a, b, c, d, v, w, x, y, and z
ions respectively


frag_mass_tolerance = 0.02


tag_search_method = 2       # 1. last
when found 2. search using the number of tag assigned by
'max_number_tag_for_search'


max_number_tags_for_search = 500  # If this number larger than the
total number of tag, the total number of tag will be used


tag_coeff_evalue = 4


##################### Dynamic Modification
######################################


# C:  57.02146 carbamidomethylation or 71.0371 acrylamide


# STY: 79.96633


# M: 15.99492


# GG: 114.04293


# SILAC K:4.02511, 6.02013, 8.01420


# SILAC R:6.02013, 10.00827


#STY = 0.000   


#K = 0.000 


dynamic_M = 15.99492


max_modif_num = 6


#################### Static Modification
############################################


add_Cterm_peptide = 0.0000


add_Nterm_peptide = 0.0000


add_G_Glycine = 0.0000


add_A_Alanine = 0.0000


add_S_Serine = 0.0000


add_P_Proline = 0.0000


add_V_Valine = 0.0000


add_T_Threonine = 0.0000


add_C_Cysteine = 57.02146


add_L_Leucine = 0.0000


add_I_Isoleucine = 0.0000


add_X_LorI = 0.0000


add_N_Asparagine = 0.0000


add_O_Ornithine = 0.0000


add_B_avg_NandD = 0.0000


add_D_Aspartic_Acid = 0.0000


add_Q_Glutamine = 0.0000


add_K_Lysine = 0.0000


add_Z_avg_QandE = 0.0000


add_E_Glutamic_Acid = 0.0000


add_M_Methionine = 0.0000


add_H_Histidine = 0.0000


add_F_Phenylalanine = 0.0000


add_R_Arginine = 0.0000


add_Y_Tyrosine = 0.0000


add_W_Tryptophan = 0.0000


add_J_user_amino_acid = 0.0000


add_U_user_amino_acid = 0.0000


#################### Cluster system
####################################################


cluster = 1


Job_Management_System = SGE


#################### Simulation ######################################################


simulation = 0   # 1 = yes; 0 = no


sim_MS1 = 0    # unit is ppm


sim_MS2 = 0     # unit is Da


###################### Output for results
############################################### 


number_of_selected_result = 3


number_of_detailed_result = 100


#####################################################################################

 
JUMP manual


*System
Requirement*


JUMP
must meet these minimum requirements.


{|  class=MsoTableGrid border=1 cellspacing=0 cellpadding=0 style='border-collapse:collapse;border:none' 
|  width=115 valign=top style='width:1.2in;border:solid windowtext 1.0pt;  background:#B6DDE8;padding:0in 5.4pt 0in 5.4pt' | 
*System*


|  width=523 valign=top style='width:5.45in;border:solid windowtext 1.0pt;  border-left:none;background:#B6DDE8;padding:0in 5.4pt 0in 5.4pt' | 
*Requirements*


|- 
|  width=115 valign=top style='width:1.2in;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Hardware


|  width=523 valign=top style='width:5.45in;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
Cluster
settings:


·        
Run on a cluster system


&Oslash; 
SGE or LSF job management system


&Oslash; 
32 GB memory on each node

 
·        
Run on a single server 


&Oslash; 
32 GB memory


&Oslash; 
2 GHz processor

 
|- 
|  width=115 valign=top style='width:1.2in;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' | 
Software


|  width=523 valign=top style='width:5.45in;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' | 
WINE
install if run .raw file


Perl
modules:

 
|- 
|  width=115 valign=top style='width:1.2in;border:solid windowtext 1.0pt;  border-top:none;padding:0in 5.4pt 0in 5.4pt' |  
|  width=523 valign=top style='width:5.45in;border-top:none;border-left:  none;border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;  padding:0in 5.4pt 0in 5.4pt' |  
|} 
*Features*


JUMP
includes the following features:

 
1.   
Support for input formats: .raw or .mzXML


2.   
Support for FASTA databases or indexed databases


3.   
 